src/esbmc/CMakeLists.txt

```diff
@@ -4,6 +4,12 @@ else()
   set(OLD_FRONTEND_TARGETS "")
 endif()
 
+if(ENABLE_SOLIDITY_FRONTEND)
+  set(SOLIDITY_FRONTEND_TARGETS solidityfrontend)
+else()
+  set(SOLIDITY_FRONTEND_TARGETS "")
+endif()
```

Comment:
I think by default the variable will be empty if it's not set. Do we need this else branch?

Reply:
Done. 
I removed the `else` block and did a few experiments - it's by default empty.